### ISSUE TITLE
Support for classic Clojure property access syntax

### ISSUE THREAD

**Issue by guv** [cursive-ide/cursive#1282]
Cursive currently insist on using the portable property access-syntax `(.-some_attribute some-deftype)`.
When using the classic syntax `(.some-attribute some-deftype)` in pure Clojure projects Cursive shows a lot of errors that `.some-attribute` cannot be resolved.

Is it possible to add optional support for the classic syntax?

**Comment by cursive-ide** [comment]
The classic syntax should work as well. Do you have a case where the portable syntax resolves correctly, but the classic does not?

**Comment by guv** [comment]
Yes, plenty of examples, otherwise I wouldn't have noticed. Taking your answer into account, I think these may be due to name "munging".
For example consider `(deftype SomeType [first-property, second-property])`.
For references of the properties as in `(let [st (SomeType. 1 2)] (.first-property st))` I get 
>  .first-property cannot be resolved
Note that the Clojure naming style with `-` is used instead of `_` which is perfectly fine for the Clojure compiler.
Maybe Clojure does not translate those property names to the proper Java ones before resolving them.

I am using (latest?) Cursive 1.1.0-15.

**Comment by cursive-ide** [comment]
Thanks for the reproduction case, I'll try this out.
